justfile

```diff
@@ -36,15 +36,15 @@ deploy-local: download-deps
 	docker run --rm -d --name cosmwasm \
 		-e PASSWORD=xxxxxxxxx \
 		-e STAKE_TOKEN=ujunox \
-		-e GAS_LIMIT=100000000 \
+		-e GAS_LIMIT=10000000 \
 		-e MAX_BYTES=22020096 \
 		-e UNSAFE_CORS=true \
 		-p 1317:1317 \
 		-p 26656:26656 \
 		-p 26657:26657 \
 		-p 9090:9090 \
 		--mount type=volume,source=junod_data,target=/root \
-		ghcr.io/cosmoscontracts/juno:v9.0.0 /opt/setup_and_run.sh {{test_addrs}}
+		ghcr.io/cosmoscontracts/juno:v11.0.0 /opt/setup_and_run.sh {{test_addrs}}
```

Comment:
We need to upgrade juno version since they made changes to wasm compile gas costs to support the max block gas lowering changes.